### ISSUE TITLE
Open new link in background tab

### ISSUE THREAD

**Issue by ByteHamster** [anthonycr/Lightning-Browser#318]
When long-pressing a link, there should be an option to open it in a new tab **and** an option to open a background tab. This way, the user can open multiple interesting websites and read them later.

**Comment by kuc** [comment]
:+1:

**Comment by anthonycr** [comment]
:+1:

**Comment by ByteHamster** [comment]
Ah, and "Open link in inkognito tag" would also be nice.

**Comment by kuc** [comment]
>Ah, and "Open link in inkognito tag" would also be nice.

This definitely would require redesigning of current popup menu for link longpress, because 5 buttons would be to much for displaying.

**Comment by anthonycr** [comment]
@kuc yes it would, if I stopped using buttons and moved to a list there's a lot of stuff I could easily add to these context menus. I'll get around to it at some point.

**Comment by tp0** [comment]
This has been fixed in 4.4.0 by:

> Improved long-press menu

https://github.com/anthonycr/Lightning-Browser/releases
